src/exec/Shelley.hs

```diff
@@ -121,9 +122,11 @@ codecs = ProtocolCodecs
   , pcLocalTxSubmissionCodec = codecLocalTxSubmission
       nodeEncodeGenTx
       nodeDecodeGenTx
-      -- Copied from cardano-node's choice.
-      Serialise.encode
-      Serialise.decode
+      -- There is at the moment no codec for UTxOValidationError
```

Comment:
My fault, but I'm working on it: https://github.com/input-output-hk/cardano-ledger/pull/598. When that has been merged, I still need to incorporate it in `ouroboros-network`, sigh.

@coot In a case like this it would be nice to make the node-to-client protocols optional.